README.md

```diff
@@ -9,11 +9,25 @@ An openvpn config file either on the openvpn server or in the inventory and an a
 
 ## Role Variables
 
-| Variable                          | Default / Mandatory        | Description                                                                                                                       |
-|-----------------------------------|----------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
-| `openvpn_client_server`           | :heavy_multiplication_x:   | Inventory name of the host running the [openvpn role](https://github.com/stuvusIT/openvpn)                                        |
-| `openvpn_client_name`             | `{{ inventory_hostname }}` | Client name used in on the openvpn server                                                                                         |
-| `openvpn_client_config_file_name` | `openvpn.ovpn`             | File name to copy over, only used when `openvpn_client_server` is not set. Looks for the file in `files/{{ inventory_hostname }}/ |
+| Variable                           | Default                    | Description                                                                             |
+| ---------------------------------- | -------------------------- | --------------------------------------------------------------------------------------- |
+| `openvpn_client_name`              | `{{ inventory_hostname }}` | Name of the client; doesn't need to match the host name                                 |
+| `openvpn_client_server`            | `default-server`           | Ansible host name of the OpenVPN server                                                 |
```

Comment:
Is `default-server` such a good default value? I find it reasonable to set a default when the role will work out-of-the-box with it, but here the user has to have a Ansible-managed server with this hostname.

Reply:
No, the server name (`openvpn_client_server`) does only need to match the Ansible-managed server hostname if `openvpn_client_local_config_path` is omitted. I added an according change to the readme.